services/main_menu_service.rb

```diff
@@ -0,0 +1,56 @@
+class MainMenuService
+  extend InputHelper
+  extend OutputHelper
+  extend Uploader
+
+  class << self
+    def main_menu(accounts, current_account)
+      loop do
+        command = ask_action(current_account.name).to_sym
+        COMMANDS.key?(command) ? send(COMMANDS[command], accounts, current_account) : wrong_command_message
+      end
+    end
+
+    private
+
+    def show_cards(_accounts, current_account)
+      CardService.show_cards(current_account)
```

Comment:
Instead of static method calls you can create instance variables of CardService, PutService, WithdrawService, SendService.
This way you can achieve something like 
```ruby
def show_cards
  @card_service.show_cards
end
```

Reply:
Also, if you do like I said and use standard library module Forwardable - you will be able to remove this methods at all and add just a couple of lines to specify delegation

Reply:
https://ruby-doc.org/stdlib-2.0.0/libdoc/forwardable/rdoc/Forwardable.html

Reply:
Thank you, I didn’t know about it. Done